.travis.yml

```diff
@@ -56,7 +60,8 @@ before_install:
 
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
-  - "travis_retry pip install -e .[all]"
+  - "travis_retry pip install -e .[$EXTRAS]"
+  - "pip freeze"
```

Comment:
🚀

Reply:
That `pip freeze` pattern you introduced is really helpful. Thanks! Just making sure we have it here too.